### DISCUSSION TITLE
Is there an ETA for version 1.19?

### DISCUSSION THREAD

**Discussion by rhodeon** [keeweb/keeweb#2180] [General Discussion]
Due to the change of the Firefox extension's UUID for keeweb-connect (https://github.com/keeweb/keeweb-connect/issues/58), the current version of Keeweb no longer works with it.
According to @HarlemSquirrel, the next release of Keeweb will fix this.
Is there currently a planned date of release for the next version? I couldn't find any mention of one in the timeline.

**Comment by rhodeon** [comment]
Closing this as I just noticed there is a discussion thread for v1.19 (https://github.com/keeweb/keeweb/discussions/2154).

**Comment by Aetherinox** [comment]
Sorry for the delay. 

Yeah Harlem has been working on the browser plugins. I'm currently in the middle of dealing with Google / Microsoft in regards to API, because some of the integrations sort of went wonky, especially Microsoft Teams which has been reported broken by a multitude of people.

But part of the delay is me taking some time to read through all the code and making sure I understand each process and what it ties into as there's a lot of code. Yesterday I had a long process of digging through the security aspects to see what direction the original developer was going in, because we've been audited in the past, and I want to ensure everything added in the future also conforms with that. And that determines which requests get approved / denied.
